entries/d/displate.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Displate": {
+    "domain": "displate.com",
+    "contact": {
+      "twitter": "Displate"
+    },
+    "keywords": [
+      "creativity",

```

Comment:
Displate isn't really a "creativity tool" as the site doesn't allow users to create their own posters.
```suggestion
```